queries.sql

```diff
@@ -24,3 +27,24 @@ SELECT * FROM animals where name <> 'Gabumon';
 --Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals where weight_kg BETWEEN 10.4 AND 17.3;
 
+/*
+  *Queries of feature 2
+*/
+-- How many animals are there?
+SELECT COUNT(*) FROM animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) FROM animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT AVG(escape_attempts) FROM animals WHERE neutered = false;
+SELECT AVG(escape_attempts) FROM animals WHERE neutered = true;
+
+-- What is the minimum and maximum weight of each type of animal?
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth > '1990-12-31' AND date_of_birth < '2000-01-01' GROUP BY species;
```

Comment:
- [ ] Kindly be noticed that you should start from `1990-01-01` to `2000-12-31`, also include equal in less than and greater than.
```sql
SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-12-31' GROUP BY species;
```